apps/course-api/src/utils/redis_utils.ts

```diff
@@ -0,0 +1,26 @@
+import { createClient } from "redis";
+
+const redisClient = createClient({
+	url: process.env.REDIS_URL
```

Comment:
ohh is this connection with redis via url?

Reply:
yup, although I saw you configure with additional stuff like `tis` `key` `cert` `ca`. Should I do the same? and can you give me the url for the site you took references from?

Reply:
actually i did it as like in the redis documentation, but i saw people do like this "url" thing too. I think it's better way since you don't really need to config stuff like port and host and password?

Reply:
well.. a url is basically put everything in one line XD

`redis://<username>:<password>@<host>:<port>`

Reply:
> yup, although I saw you configure with additional stuff like `tls` `key` `cert` `ca`. Should I do the same? and can you give me the url for the site you took references from?

Isn't the TLS, Cert, CA and Key thing for SSL? If the service needs to go through SSL, wouldn't it need that configuration?

Reply:
Does it need to go through SSL? It's an internal docker network anyway though

`redis://USERNAME:PASSWORD@redis:6379`